ibm/mas_devops/roles/dro/tasks/main.yml

```diff
@@ -29,7 +29,14 @@
 - name: UDS to DRO migration (mas update scenario)
   include_tasks: "tasks/install-dro/main.yml"
   when:
-    - dro_migration != "" and dro_migration | lower == "true"
+    - dro_migration != ""
+    - dro_migration | lower == "true"
+  block:
+    - set_fact:
+        migration_status: "installing-dro"
+
+    - name: install dro
+      include_tasks: "tasks/install-dro/main.yml"
```

Comment:
We are calling install-dro/main.yml twice?

Reply:
`install-dro` is called only once depending on the dro_migration value.

Reply:
Are you sure? 
![image](https://github.com/ibm-mas/ansible-devops/assets/6817894/6b3f84d8-639f-474b-bf51-03a4fad06747)

Reply:
ohh. my bad, removed that block as it is not required.